pkg/ingester/ingester.go

```diff
@@ -859,19 +852,22 @@ func (i *Ingester) PushWithCleanup(ctx context.Context, req *mimirpb.WriteReques
 	// At this point all samples have been added to the appender, so we can track the time it took.
 	i.metrics.appenderAddDuration.Observe(time.Since(startAppend).Seconds())
 
-	if span != nil {
-		span.LogFields(otlog.String("event", "start commit"),
-			otlog.Int("succeededSamplesCount", succeededSamplesCount),
-			otlog.Int("failedSamplesCount", failedSamplesCount),
-			otlog.Int("succeededExemplarsCount", succeededExemplarsCount),
-			otlog.Int("failedExemplarsCount", failedExemplarsCount))
-	}
+	level.Debug(spanlog).Log(
+		"event", "start commit",
+		"succeededSamplesCount", succeededSamplesCount,
+		"failedSamplesCount", failedSamplesCount,
+		"succeededExemplarsCount", succeededExemplarsCount,
+		"failedExemplarsCount", failedExemplarsCount,
+	)
 
 	startCommit := time.Now()
 	if err := app.Commit(); err != nil {
 		return nil, wrapWithUser(err, userID)
 	}
-	i.metrics.appenderCommitDuration.Observe(time.Since(startCommit).Seconds())
+
+	commitDuration := time.Since(startCommit)
+	i.metrics.appenderCommitDuration.Observe(commitDuration.Seconds())
+	level.Debug(spanlog).Log("event", "complete commit", "commitDurationSeconds", commitDuration.Seconds())
```

Comment:
nit:
```suggestion
	level.Debug(spanlog).Log("event", "complete commit", "commitDuration", commitDuration.String())
```

these seem to be in the order of `0.00955` seconds, so it might be easier to pretty-print it

Reply:
Good idea!